GermanFilter/sections/specific.txt

```diff
@@ -115,7 +115,7 @@ vergleich.focus.de,chip.de,gutefrage.net,reisefrage.net,motorradfrage.net,autofr
 ! ##.adsbygoogle
 winfuture.de,netzwelt.de,quoka.de,farmeramania.de,bravofly.de,computerbild.de,digitalfernsehen.de,hl-live.de,chinamobilemag.de,focus.de,pcwelt.de##.adsbygoogle
 !
-tagesspiegel.de##.Wdy.BNw
+tagesspiegel.de##div[class^="Wd"]
```

Comment:
@Morku let's specify this rule a bit. Please check this on your end
```
tagesspiegel.de##main div[class^="Wd"]
```

Reply:
yes, it's also working fine :)